src/pages/search-location/SearchLocationDaum.tsx

```diff
@@ -25,47 +26,58 @@ const THEME_OBJ = {
 const SearchLocationDaum = () => {
   const daumLocationSearchRef = useRef<HTMLDivElement>(null);
   const router = useRouter();
-  const setLocation = useSetReastorage(locationStorage);
+  const { getPosition } = useSetLocation();
   const { nextUrl } = router.query as { [key: string]: string };
-  const [isScriptLoading, setIsScriptLoading] = useState(false);
+  const [isScriptLoading, setIsScriptLoading] = useState({
+    daum: false,
+    naver: false,
+  });
 
   const { width, height } = useWindowSize();
   const handleBack = () => {
     router.back();
   };
 
   const handleDaumPostcode = () => {
-    setIsScriptLoading(true);
+    setIsScriptLoading((prev) => ({ ...prev, daum: true }));
+  };
+  const handleNaverGeocode = () => {
+    setIsScriptLoading((prev) => ({ ...prev, naver: true }));
   };
```

Comment:
```suggestion
const handleLoadEnd = (key: keyof typeof isScriptLoading) => () => {
  setIsScriptLoading((prev) => ({ ...prev, [key]: true }));
}
```
이렇게 쓰면 함수 하나로 줄일 수 있을것 같아요~